### ISSUE TITLE
metadata problem when calling isset()

### ISSUE THREAD

**Issue by flack** [midgardproject/midgard-php5#26]
With today's dev packages from OBS, the following code

```
$t = new midgard_topic('3a31fa768c6911db8866852cac19ea14ea14');
var_dump(isset($t->metadata));
var_dump(property_exists($t->metadata, 'schedulestart'));
```

produces this output:

```
boolean true

( ! ) Warning: First parameter must either be an object or the name of an existing class in file.php on line 29
Call Stack
#   Time    Memory  Function    Location
1   0.0007  81764   {main}( )   ../rootfile.php:0
2   0.0068  85928   property_exists ( ) ../rootfile.php:29

null
```

(Disclaimer: This is probably related to issue 25, but I don't know if gh allows reopening tickets, so I just created a new one)

**Comment by indeyets** [comment]
flack: dev-packages from OBS provide recent midgard-core, but midgard-php5 is terribly outdated. you'll need to build midgard-php5 manually, until piotras fixes this.

I believe this bug will go away, if you use recent midgard-php5

**Comment by flack** [comment]
just tried to install from source, but it wouldn't let me. pake install aborts with 

```
configure: error: Midgard2 version 10.05.1 or greater required. 
```

which seems logical, given that 

```
pkg-config --list-all | grep midgard
```

returns
    midgard                        Midgard - Midgard Framework Library (1.x)

So does that mean that I can only compile it on a machine that doesn't have mgd1 installed or is this rather a bug in the mgd2 core packages (i.e. should pkg-config also list a pakcage midgard2)?

**Comment by indeyets** [comment]
pkg-config is supposed to list midgard2 separately. I guess you need to install libmidgard2-dev package from OBS

**Comment by flack** [comment]
I installed the latest ratatoskr from git now, stopped and started lighttpd, but the error mentioned above stays the same

**Comment by indeyets** [comment]
thanks. will check it

**Comment by indeyets** [comment]
fixed in 957450dbada23f990e1e561eddf85ee1e8f7d501
